### ISSUE TITLE
Team discussions with outside collaborators

### ISSUE THREAD

**Issue by jparise** [todogroup/gh-issues#58]
[Team discussions](https://help.github.com/articles/about-team-discussions) are handy, but they would be even more useful if [outside collaborators](https://help.github.com/articles/adding-outside-collaborators-to-repositories-in-your-organization/) could also participate in them.

**Comment by caniszczyk** [comment]
@jparise check out this feature https://github.com/vercel/next.js/discussions which is a new github feature, does that help?

**Comment by jparise** [comment]
Yes, it helps for some cases. I was originally thinking it would be good to have a private place to talk amongst just the folks who can commit to a project, though. In other words, there's the organizational "team" and there's the "committers", which is "team + outside collaborators".
